examples/src/utils/example-helpers.ts

```diff
@@ -2,37 +2,50 @@
  * Copyright 2020 Cognite AS
  */
 
-import { RevealManager, LocalHostRevealManager } from '@cognite/reveal/experimental';
-import { CogniteClient } from '@cognite/sdk';
-import { RevealOptions } from '@cognite/reveal/public/RevealManagerBase';
+import {
+  RevealManager,
+  LocalHostRevealManager,
+} from "@cognite/reveal/experimental";
+import { CogniteClient } from "@cognite/sdk";
+import { RevealOptions } from "@cognite/reveal/public/RevealManagerBase";
 
 // TODO 22-05-2020 j-bjorne: change to return render and added models
 export function createRenderManager(
-  type: 'local' | 'cdf',
+  type: "local" | "cdf",
   client: CogniteClient,
   options?: RevealOptions
 ): RevealManager | LocalHostRevealManager {
   switch (type) {
-    case 'cdf':
+    case "cdf":
       return new RevealManager(client, options);
-    case 'local':
+    case "local":
```

Comment:
Will there not be a lint error here? Are we all using the same linter?

Reply:
examples have their one eslint installed inside react-scripts, formatting is done by prettier. I believe vscode is also should understand installed prettier and format accordingly. Please let me know if there are some issues with that.